app/javascript/packs/public.js

```diff
@@ -3,6 +3,7 @@ import { getLocale } from 'mastodon/locales';
 import { length } from 'stringz';
 import IntlRelativeFormat from 'intl-relativeformat';
 import { delegate } from 'rails-ujs';
+import loadPolyfills from '../mastodon/load_polyfills';
```

Comment:
`mastodon` isn't an npm package (well actually [it is](https://www.npmjs.com/package/mastodon) 😅 ) so I don't think that's what we want…